pywinauto/unittests/test_atspi_controls.py

```diff
@@ -89,32 +83,22 @@ def test_button_click(self):
             self.assertEqual(self._get_state_label_text(), "\"Click\" clicked")
 
         def test_button_toggle(self):
-            # TODO replace .children call to wrapper object when wrapper fully implemented
-            toggle_button_info = self.app_info.children()[0].children()[0].children()[3]
-            toggle_button_wrapper = AtspiWrapper(toggle_button_info)
+            toggle_button_wrapper = self.app_wrapper.Frame.Panel.Button
```

Comment:
If you name this variable `*_wrapper`, let's call `.wrapper_object()` for the window specification. It will speed the tests up because search is done only once.